docker/mlpack_cli/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:latest
```

Comment:
Another option would be to save a lot of code by making a "base" mlpack image that has all dependencies required for the core library, and then use that base as the `FROM`.  This means less code to modify for each individual binding type, too.